samples/multicore/empty/README.rst

```diff
@@ -1,14 +1,14 @@
 .. _nrf5340_empty_net_core:
 
-nRF5340: Empty firmware for network core
-########################################
+Empty firmware for network (or other) core
+##########################################
 
 .. contents::
    :local:
    :depth: 2
 
-The sample demonstrates how to generate an empty network core firmware.
-The sample is used only by the applications that do not use the network core.
+The sample demonstrates how to generate an empty network (or other) core firmware.
```

Comment:
line no:24 needs to be updated with correct sample file path - :file:`samples/multicore/empty`